packages/cli/src/lib/Manifest.ts

```diff
@@ -14,9 +14,12 @@ interface ClientModule {
   module: ModulePath;
 }
 
+type ManifestVersion = string;
```

Comment:
Why is this useful?

Reply:
Testing purposes at first, it is not useful at all. Will remove